integration/conftest.py

```diff
@@ -101,10 +108,48 @@ def cleanup():
     return tmp
 
 
+# NB: conceptually, it kind of makes sense to parametrize this fixture
+# on "Tahoe version" .. a quick attempt at that lead to all but the
+# first version failing; I think something doesn't get cleaned-up
+# properly between runs "or something". Instead, we use a command-line
+# option to specify tahoe-version and have multiple separate runs
+# (which looks better in CI anyway).
+@pytest.fixture(
+    scope='session',
+)
+def tahoe_venv(request, reactor, temp_dir):
+    """
+    A virtualenv for our Tahoe install, letting us install a different
+    one from the Tahoe we depend on.
+    """
+    venv_dir = join(temp_dir, "tahoe_venv")
+    tahoe_version = request.config.getoption("tahoe_version")
+    print("creating venv", venv_dir, sys.executable)
+    out_protocol = _DumpOutputProtocol(None)
+    reactor.spawnProcess(
+        out_protocol,
+        sys.executable,
+        ("python", "-m", "virtualenv", venv_dir),
+        env=environ,
+    )
+    pytest_twisted.blockon(out_protocol.done)
```

Comment:
Any reason we can't just use `subprocess.check_call` here?

Reply:
Probably not? I guess "consistency with other calls"?